### PR TITLE
UI - menu items in business tombstone

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/21785
*Issue:*https://github.com/bcgov/entity/issues/23422

*Description of changes:*
- Dissolve this Business - open the confirmation dialog
- Consent to Amalgamate Out - redirect
- Consent to Continue Out - redirect
- Request AGM Extension - redirect
- Request AGM Location Change - redirect
- Amalgamate - redirect

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-48-e95gc34d.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-48-e95gc34d.web.app

File: src/components/bcros/businessDetails/Links.vue [hunk 1]
```diff
@@ -41,10 +42,50 @@ const isChangeBusinessInfoDisabled = computed(() => {
   return !isAllowed
 })
 
+const legalName = computed(() => currentBusiness.value?.legalName)
+// TO-DO: in the old codebase, the legalName is returned by the getLegalName() function.
+// Need to investigate the logic and implement it in business store.
```
Comment (hfekete) [inline src/components/bcros/businessDetails/Links.vue:47]: Can we create a ticket if it's not going to be done as part of  this ticket and then add new ticket number in the comment, thanks !

File: src/composables/useBcrosNavigate.ts [hunk 2]
```diff
@@ -41,10 +41,22 @@ export const useBcrosNavigate = () => {
   function goToSetupAccount () {
     redirect(config.public.authWebURL + 'setup-account')
   }
-  const goToDigitalCredentialsPage = () => {
+  function goToDigitalCredentialsPage () {
```
Comment (hfekete) [inline src/composables/useBcrosNavigate.ts:44]: What is the reason to converting it to this type of declaration, is it to keep it the same as other function definitions in the file?

**Review by hfekete — APPROVED** [review]
Good work! 
There is one question in comment, and one comment for creating ticket and adding its ticket number in the code comment.

File: src/composables/useBcrosNavigate.ts (hunk 2 above)
Comment (patrickpeinanw) [reply src/composables/useBcrosNavigate.ts:44]: Yeah it is just to keep the declaration consistent within the file.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-48-e95gc34d.web.app

**Pull request merged**